core/src/main/scala/kafka/consumer/ZookeeperConsumerConnector.scala

```diff
@@ -111,6 +111,16 @@ private[kafka] class ZookeeperConsumerConnector(val config: ConsumerConfig,
   private val zkCommitMeter = newMeter("ZooKeeperCommitsPerSec", "commits", TimeUnit.SECONDS, Map("clientId" -> config.clientId))
   private val rebalanceTimer = new KafkaTimer(newTimer("RebalanceRateAndTime", TimeUnit.MILLISECONDS, TimeUnit.SECONDS, Map("clientId" -> config.clientId)))
 
+  newGauge(
+    "yammer-metrics-total",
+    new Gauge[Int] {
+      def value = {
+        import scala.collection.JavaConverters._
+        com.yammer.metrics.Metrics.defaultRegistry().groupedMetrics().asScala.foldLeft(0)(_ + _._2.size())
```

Comment:
One more minor naming thing I thought of later: `yammer-metrics-count` and `kafka-metrics-count` sound a little better.